Makefile

```diff
@@ -194,6 +194,21 @@ antctl: $(ANTCTL_BINARIES)
 antctl-release:
 	@$(GO) build -o $(BINDIR)/$(ANTCTL_BINARY_NAME) $(GOFLAGS) -ldflags '-s -w $(LDFLAGS)' antrea.io/antrea/cmd/antctl
 
+.PHONY: check-copyright
+check-copyright: 
+	@echo "==> Installing addlicense <=="
+	@$(GO) install github.com/google/addlicense@latest
+	@echo "===> Checking License for files <==="
+	addlicense -c "Antrea Authors." -check `find . -type f -name "*.go"` `find . -type f -name "*.sh"`
+	@echo "Have checked out files without copyright, if you want to add them, please run 'make add-copyright'"
```

Comment:
If the above command fails, this message is not printed. I don't think it was your intention.

Reply:
> If the above command fails, this message is not printed. I don't think it was your intention.

changed

Reply:
I changed it back to if the addlicense line fails, the next message will be blocked principle. Because it is weird that if we failed on checking the files without copyright, then we give a message said "Have checked out files without copyright".